pro/src/routes/OfferEducationalStockCreation/OfferEducationalStockCreation.tsx

```diff
@@ -4,7 +4,8 @@ import { useParams } from 'react-router-dom'
 
 import useNotification from 'components/hooks/useNotification'
 import Spinner from 'components/layout/Spinner'
-import { getOfferAdapter } from 'core/OfferEducational'
+import { getOfferAdapter, Mode } from 'core/OfferEducational'
+import { DEFAULT_EAC_STOCK_FORM_VALUES } from 'core/OfferEducationalStock/constants'
 import { OfferEducationalStockFormValues } from 'core/OfferEducationalStock/types'
```

Comment:
tout devrait être exporté depuis l'index : 

```suggestion
import {
  getOfferAdapter,
  Mode,
  DEFAULT_EAC_STOCK_FORM_VALUES,
  OfferEducationalStockFormValues
} from 'core/OfferEducational'
```